### ISSUE TITLE
Feed results back into global admin data package

### ISSUE THREAD

**Issue by Robinlovelace** [afrimapr/afriadmin#7]
Following on from discussion today, I think it would be great if the results go into 'metapackages' to improve open access to admin data worldwide. Opening more as a point of conversation than anything else, but think it's worth thinking early about how this could feed into global efforts.

**Comment by andysouth** [comment]
I agree, good to keep in mind how anything that we get to work for Africa can later be applied to rest of world. Also from our discussion that anything that is useful more generally be put in a more general package(s).
